sklearn/calibration.py

```diff
@@ -19,6 +20,8 @@
 from scipy.special import xlogy
 from scipy.optimize import fmin_bfgs
 
+from .utils._param_validation import StrOptions
```

Comment:
Can you move this entry below the line `from .utils.fixes import delayed`.

Also you need to import the following:

```suggestion
from .utils._param_validation import HasMethods, StrOptions
```